### ISSUE TITLE
mongodb.so for php 5.4-7.0 not available

### ISSUE THREAD

**Issue by tomzx** [travis-ci/travis-ci#5621]
See https://travis-ci.org/tomzx/github-repository-comparator/builds/108947649 for example.

```
PHP Warning:  PHP Startup: Unable to load dynamic library '/home/travis/.phpenv/versions/5.6.5/lib/php/extensions/no-debug-zts-20131226/mongodb.so' - /home/travis/.phpenv/versions/5.6.5/lib/php/extensions/no-debug-zts-20131226/mongodb.so: cannot open shared object file: No such file or directory in Unknown on line 0
```

**Comment by BanzaiMan** [comment]
Could you test `7.0snapshot` and `nightly` builds?

**Comment by BanzaiMan** [comment]
For older PHP runtimes, I don't think we built `mongodb`. Does `mongo` work?

**Comment by tomzx** [comment]
@BanzaiMan I don't mind mongodb not being available for older release, but if that is the case, I'd suggest updating https://docs.travis-ci.com/user/languages/php#PHP-5.6-and-below which says:

> PHP 5.6 and below
> For PHP versions up to 5.6, these extensions are available.
> [...]
> mongo.so or mongodb.so

Let me know if I should fill an issue in the documentation repository.

I've built `7.0snapshot` and `nightly` as requested @ https://travis-ci.org/tomzx/github-repository-comparator/builds/109510344. Both appear to have mongodb.so available.

As a final note, the MongoDB extension (mongo.so) (http://php.net/manual/en/book.mongo.php) is now legacy and the php documentation suggests to use the newer extension (mongodb.so).

Based on http://php.net/manual/en/mongodb.requirements.php, it should be possible to make mongodb.so available for version 5.4 and up.

**Comment by BanzaiMan** [comment]
I agree. Documentation needs updated.
